components/compliance-service/ingest/ingestic/mappings/comp-rep-date.go

```diff
@@ -408,6 +408,9 @@ var ComplianceRepDate = Mapping{
             "skip_message": {
               "type": "keyword"
             },
+            "status_message": {
```

Comment:
we're going to need to bump the compliance index version in mappings.go

Reply:
wait what? i thought this was a new field only and as such doesn’t require a migration? we need to talk to support if we’re doing a migration..

Reply:
> by not specifying, status_message will default to type "text"

We are specifying it in the mapping here and the structs. Earlier Automate versions will get this `status_message` but without data collector extraction and existance in the go structs, the field will not reach ElasticSearch.